dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -50,7 +51,9 @@
   private static final Logger logger = Logger.getLogger(DependencyGraphBuilder.class.getName());
   
   private static final RepositorySystem system = RepositoryUtility.newRepositorySystem();
-  
+
+  private static final Pattern NON_ALPHA_NUMERIC_PATTERN = Pattern.compile("[^a-z0-9]+");
```

Comment:
Tiny nit: You don't need to have "_PATTERN" since we know the type.

Reply:
Removed _PATTERN